README.md

```diff
@@ -4,97 +4,164 @@
-      * [Install specific nvm version](#install-specific-nvm-version)
-      * [Miscellaneous](#miscellaneous)
-      * [Run in a container](#run-in-a-container)
-         * [buildah/podman](#buildahpodman)
-            * [Building the container](#building-the-container)
-            * [Running the container](#running-the-container)
-            * [k8s](#k8s)
-         * [Docker](#docker)
-            * [Build with docker-compose](#build-with-docker-compose)
-            * [Run with docker](#run-with-docker)
+      * [Introduction](#introduction)
+      * [Available commands](#available-commands)
+      * [Configuration](#configuration)
+      * [Running the client locally](#running-the-client-locally)
+      * [Running the client in a container](#running-the-client-in-a-container)
+      * [Available npm scripts](#available-npm-scripts)
       * [References](#references)
 
 ## Introduction
 
-Contains automation used by the snowdrop team
+This project implements automation commands used by the Snowdrop team.
+The commands are executed with Slack messages and integrate with Atomist SDM.
 
-## Basics
+This application can run in any environment that can run a Node.js application.
+Including a server in a private network.
 
-The automation client is a NodeJS application that communicates with the Atomist API in order to
-perform operations an receive events.
 
-This application can anywhere a NodeJS application can run. One thing to note is that it can also
-run inside a private network since it uses Websockets to communicate with the Atomist API.
+## Available commands
 
-## Running
+### Update an example property
 
-### Prerequisites
+This command will update a `pom.xml` property of all of the Snowdrop example applications.
 
-* Properly configure your environment in order for Atomist to be able to work. See [this](https://docs.atomist.com/developer/prerequisites/)
+Send the following Slack message to Atomist:
+```
+update boosters maven property
+```
+
+You will need to provide a git branch, property name and a new value of the property.
+
+
+### Create a launcher pull request
 
-Of particular importance is the `Minimal Configuration` section which mentions that at the very least, a file named `$HOME/.atomist/client.config.json`
-is needed containing the following information
+This command updates our fork of the Launcher catalog and raises a pull request upstream.
 
+Send the following Slack message to Atomist:
 ```
-{
-  "apiKey": "API_KEY",
-  "workspaceIds": [
-    "WORKSPACE_ID"
-  ]
-}
+create launcher pr
 ```
 
-with API_KEY and WORKSPACE_ID replaced with your Atomist API key and workspace ID, respectively. 
+You will need to provide a Spring Boot version that should be used by the Launcher (e.g. 2.3.0.RELEASE). The script will figure out which example tags to use based on that version and will update the Launcher accordingly.
 
-* Node 8.x or higher
+### Update an example
 
-Install dependencies by executing:
+TODO
```

Comment:
?

Reply:
I skipped this one, because I have never used it and didn't feel competent enough to document it. I'll do this one separately another time.